packages/smooth_ui_library/lib/widgets/smooth_expandable_card.dart

```diff
@@ -1,10 +1,9 @@
+import 'dart:math';
+
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
-import 'package:provider/provider.dart';
-import 'package:smooth_ui_library/widgets/models/single_boolean_model.dart';
-import 'package:smooth_ui_library/widgets/smooth_card.dart';
 
-class SmoothExpandableCard extends StatelessWidget {
+class SmoothExpandableCard extends StatefulWidget {
   const SmoothExpandableCard({
     @required this.collapsedHeader,
     this.expandedHeader,
```

Comment:
We may need a `bool collapsed` in the constructor as a default value; there might already be an issue about that.

Reply:
I don't know if I understood you exactly, do you mean that we can set the startvalue for each widget inindividually, or something else